nouvelles-perspectives-des-sciences-sociales.csl

```diff
@@ -0,0 +1,606 @@
+      <term name="in">dans </term>
+      <term name="edition">édition</term>
+      <term name="accessed">consulté le</term>
+      <term name="at">disponible sur :</term>
+      <term name="et-al">et al.</term>
+      <term name="ibid" form="short">ibid</term>
+      <term name="issue" form="short">nᵒ</term>
+      <term name="collection-number" form="short">nᵒ</term>
+      <term name="chapter-number" form="short">chap.</term>
+    </terms>
+  </locale>
+  <macro name="contributors">
+    <choose>
+      <if type="song" match="all" variable="composer">
+        <names variable="composer">
+          <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="all"/>
+        </names>
+      </if>
+      <else>
+        <names variable="author">
+          <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="all">
+            <name-part name="family"/>
+          </name>
+          <substitute>
+            <choose>
+              <if type="webpage">
+                <text macro="title"/>
+              </if>
+              <else>
+                <text macro="editor"/>
+                <text macro="title"/>
+              </else>
+            </choose>
+          </substitute>
+        </names>
+      </else>
+    </choose>
+  </macro>
+  <macro name="contributors-notes">
+    <choose>
+      <if type="song" match="all" variable="composer">
+        <names variable="composer">
+          <name and="text" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="3" initialize="false"/>
+        </names>
+      </if>
+      <else>
+        <names variable="author composer">
+          <name and="text" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="3" initialize="false"/>
+          <substitute>
+            <choose>
+              <if type="webpage">
+                <text macro="title"/>
+              </if>
+              <else>
+                <text macro="editor-notes"/>
+                <text macro="title"/>
+              </else>
+            </choose>
+          </substitute>
+        </names>
+      </else>
+    </choose>
+  </macro>
+  <macro name="contributors-sort">
+    <choose>
+      <if type="song" match="all" variable="composer">
+        <names variable="composer"/>
+      </if>
+      <else>
+        <names variable="author editor">
+          <substitute>
+            <text macro="title"/>
+            <text macro="container-title"/>
+          </substitute>
+        </names>
+      </else>
+    </choose>
+  </macro>
+  <macro name="translator">
+    <names variable="translator">
+      <label form="verb" suffix=" "/>
+      <name and="text" delimiter-precedes-last="never" initialize="false" sort-separator=" "/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <group>
+      <names variable="editor" delimiter=", ">
+        <name and="text" delimiter-precedes-last="never" initialize="false" name-as-sort-order="all">
```

Comment:
no need for initialize="false" without `initialize-with`

Reply:
OK

Reply:
This modification will not affect our proposition ? We need a bibliography for our journal as the sample citations that csl-bot sent us. The citations generated and based on our proposed changes.

Reply:
It shouldn't, no.

That said, if you discover you need to make any changes to the style, please follow the instructions at https://github.com/citation-style-language/styles/blob/master/CONTRIBUTING.md#3b-submitting-changes-to-an-existing-style.